Source/Shaders/ModelExperimental/GeometryVS.glsl

```diff
@@ -0,0 +1,32 @@
+void processPoints() {
```

Comment:
@sanjeetsuhag one thing we both should note is our Coding Guide https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#formatting-1 uses 4-space tabs and braces on their own line.